nixos/modules/virtualisation/qemu-vm.nix

```diff
@@ -868,6 +880,17 @@ in
           '';
       };
 
+    virtualisation.useHostCerts =
+      mkOption {
+        type = types.bool;
+        default = false;
+        description =
+          lib.mdDoc ''
+            If enabled, always use `NIX_SSL_CERT_FILE` from the host
+            to pass CA certificates to the VM.
+          '';
+      };
```

Comment:
Could we accept an optional path to a certificate file instead? This way `qemu-vm` need not implicitly depend upon NIX_SSL_CERT_FILE.